.eslintrc.js

```diff
@@ -11,7 +11,12 @@ module.exports = {
   rules: {
     'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
     'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
-    'max-len': ['warn', { code: 120, ignoreRegExpLiterals: true }]
+    'max-len': ['warn', { code: 120, ignoreRegExpLiterals: true }],
+    // 'vue/no-deprecated-filter': 'warn',
+    // 'vue/no-deprecated-v-on-native-modifier': 'warn',
```

Comment:
These 2 rules don't exist in ESLint that is compatible with Vue 2.7, but they will exist with Vue3.